addons/pos_loyalty/models/loyalty_program.py

```diff
@@ -7,7 +7,9 @@
 class LoyaltyProgram(models.Model):
     _inherit = 'loyalty.program'
 
-    pos_config_ids = fields.Many2many('pos.config', string="Point of Sales", readonly=True)
+    # NOTE: `pos_config_ids` satisfies an excpeptional use case: when no PoS is specified, the loyalty program is
+    # applied to every PoS. You can access the loyalty programs of a PoS using _get_program_ids() of pos.config
+    pos_config_ids = fields.Many2many('pos.config', relation='pos_config_loyalty_program_rel', column1='loyalty_program_id', column2='pos_config_id')
```

Comment:
I don't see the need for specifying column1 and column2. Are they important in the migration?